solvebio/resource/object.py

```diff
@@ -0,0 +1,69 @@
+"""Solvebio Object API resource"""
+from ..help import open_help
+from ..client import client
+
+from .solveobject import convert_to_solve_object
+from .apiresource import CreateableAPIResource
+from .apiresource import ListableAPIResource
+from .apiresource import SearchableAPIResource
+from .apiresource import UpdateableAPIResource
+from .apiresource import DeletableAPIResource
+
+
+class Object(CreateableAPIResource,
```

Comment:
Currently the client seems very focused on I/O operations => get data into SolveBio, take it out for additional analysis.  Doing things in the client that are possible in the UI doesn't feel especially necessary yet.  If you feel strongly that I should add these, I will, but otherwise my vote is to get the Client released in its more-or-less current form, get the customers upgraded, and then come back to the Client based on additional customer feedback and our own internal feedback.